server/config/routes.js

```diff
@@ -378,12 +379,19 @@ exports.configure = function (app) {
   app.get('/patients/search/name/:value', patient.searchFuzzy);
   app.get('/patients/search/reference/:value', patient.searchReference);
 
-  // Debtors API
+  /** Debtors API */
+  /** @fixme `/debtors/groups` is deprecated, use `/debtor_groups` at the client side */
```

Comment:
For future reference, there is a [@deprecated](http://usejsdoc.org/tags-deprecated.html) tag in JSDoc.  Pretty cool, huh?